examples/dna_language_models/dna_lm.ipynb

```diff
@@ -0,0 +1,1369 @@
+    "training_args = TrainingArguments(\n",
+    "    output_dir='./results',\n",
+    "    evaluation_strategy=\"epoch\",\n",
+    "    learning_rate=2e-5,\n",
+    "    per_device_train_batch_size=16,\n",
+    "    per_device_eval_batch_size=16,\n",
+    "    num_train_epochs=5,\n",
+    "    weight_decay=0.01,\n",
+    ")\n",
+    "\n",
+    "# Initialize Trainer\n",
+    "trainer = Trainer(\n",
+    "    model=classification_model,\n",
+    "    args=training_args,\n",
+    "    train_dataset=train_dataset,\n",
+    "    eval_dataset=val_dataset,\n",
+    "    tokenizer=tokenizer,\n",
+    "    data_collator=data_collator,\n",
+    ")\n",
+    "\n",
+    "# Train the model\n",
+    "trainer.train()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ebc7e33a-caad-4412-84e3-3e1ce7d02ccd",
+   "metadata": {},
+   "source": [
+    "### 5. Evaluation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 104,
+   "id": "38eb0273-ce7e-4770-8457-2f9609f6843b",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[1 1 1 1 0 1 1 0 0 0 0 1 1 1 1 0 1 1 0 0 1 1 0 1 0 0 1 0 0 0 1 1 1 0 0 0 0\n",
+      " 1 1 0 0 0 1 1 0 1 0 0 0 0 0 1 1 1 0 0 1 1 1 0 1 1 1 1 0 0 1 0 1 1 0 0 0 1\n",
+      " 0 0 0 1 1 0 1 0 0 1 0 0 1 1 1 0 0 0 0 1 1 1 0 1 0 0 1 1 0 0 0 1 0 0 0 0 1\n",
+      " 0 0 1 1 0 0 1 1 0 0 0 0 1 1 1 1 1 0 1 0 0 1 0 1 1 1 0 1 1 0 1 0 1 1 0 0 1\n",
+      " 0 0 1 1 1 0 1 0 1 0 1 1 1 1 1 1 1 1 0 1 1 1 0 1 0 0 0 1 1 1 1 1 1 0 1 0 0\n",
+      " 1 1 0 1 1 0 0 1 1 1 0 1 1 1 1]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Generate predictions\n",
+    "\n",
+    "predictions = trainer.predict(test_dataset)\n",
+    "logits = predictions.predictions\n",
+    "predicted_labels = logits.argmax(axis=-1)\n",
+    "print(predicted_labels)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 105,
+   "id": "327a1c3b-88d6-4430-8978-73a7cbdbb697",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Accuracy: 0.795\n",
+      "Precision: 0.794957729468599\n",
+      "Recall: 0.795\n",
+      "F1-Score: 0.794799276945093\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Step 3: Evaluate the Predictions (Optional)\n",
+    "from sklearn.metrics import accuracy_score, precision_recall_fscore_support\n",
+    "\n",
+    "accuracy = accuracy_score(test_labels, predicted_labels)\n",
+    "print(f\"Accuracy: {accuracy}\")\n",
+    "\n",
+    "precision, recall, f1, _ = precision_recall_fscore_support(test_labels, predicted_labels, average='weighted')\n",
+    "print(f\"Precision: {precision}\")\n",
+    "print(f\"Recall: {recall}\")\n",
+    "print(f\"F1-Score: {f1}\")"
```

Comment:
Precision/recall/F1 are all the same and identical to accuracy. I wonder if it's not enough to just report accuracy? As accuracy can be calculated easily, this would also allow us to remove the sklearn dependency, which is also not a PEFT dependency and would otherwise require users to install it.

Reply:
Yes, changed!